.fixtures.yml

```diff
@@ -13,6 +13,7 @@ fixtures:
     foreman:       "git://github.com/theforeman/puppet-foreman"
```

Comment:
Just Foreman or all of them? I was referencing auditing the list as a whole (e.g. tftp, xinetd, dhcp, dns, foreman_proxy etc.)